features/canvas.yml

```diff
@@ -1,15 +1,16 @@
-draft_date: 2024-06-05
-name: <canvas>
-description: The `<canvas>` element TODO.
+name: Canvas API
+description: The canvas API provides a means for drawing graphics via JavaScript and the HTML `<canvas>` element.
 spec: https://html.spec.whatwg.org/multipage/canvas.html#canvas
 group: html
 compat_features:
```

Comment:
@petele can you add `compute_from: html.elements.canvas` in a `status` block here to get the computed status for the feature to be correct?

Reply:
done